src/main/java/dk/bec/polonez/reservationsystem/dto/offerDto/CreateOfferDto.java

```diff
@@ -8,4 +8,6 @@
     private String name;
 
     private String description;
+
+    private long ownerId;
```

Comment:
**advice** The package of this class is incorrect when it comes to Java conventions (upper-case should bever be used in package names):

```dk.bec.polonez.reservationsystem.dto.offerDto``` -> ```dk.bec.polonez.reservationsystem.dto.offer```